test/bigtest/tests/useCustomFields-test.js

```diff
@@ -57,7 +64,7 @@ const doLogout = async function () {
   await i.clickLogout();
 };
 
-describe('useCustomFields hook', () => {
+describe.only('useCustomFields hook', () => {
```

Comment:
Are you sure that it should be the only test?

Reply:
Thanks for noticing!